server/utils/validation/historyLogValidation/index.js

```diff
@@ -0,0 +1,10 @@
+const { object, number, string } = require('yup');
+
+const historyLogSchema = object({
+  patientId: number().min(0).required(),
+  description: string().max(200).required(),
```

Comment:
I think limiting the description with 200 characters only is a bad thing

Reply:
done